src/java/org/apache/cassandra/journal/Journal.java

```diff
@@ -510,6 +536,25 @@ public RecordPointer asyncWrite(K id, V record, Set<Integer> hosts)
         return recordPointer;
     }
 
+    public RecordPointer asyncWrite(K id, Writer writer, Set<Integer> hosts)
```

Comment:
this looks like copy/paste of `org.apache.cassandra.journal.Journal#asyncWrite`. should we refactor?